### PR TITLE
Fix dangling comma in SSO signup form, minor language changes

### PR DESCRIPTION
**Pull request by david-code**
## Checklist

1. [x] If you've added code that should be tested, add tests
2. [x] If you've changed APIs, update (or create!) the documentation
3. [x] Ensure the tests pass
4. [x] Make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/master/CONTRIBUTING.md)
5. [x] Write a description of your work suitable for publishing on [our forum](https://community.kobotoolbox.org/tag/release-notes)
6. [x] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)
7. [x] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes

## Description

Update SSO signup form to fix hanging comma when user's SSO account username not available. Minor language changes 

## Additional Comments

I'm not sure if we need to update the translations here. I seem to remember there being some documentation about this, but I can't find it now.

@bufke

### CODE REVIEW

File: kobo/apps/accounts/templates/socialaccount/signup.html [hunk 1]
```diff
@@ -7,7 +7,11 @@
 
 <form class="registration registration--social" id="signup_form" method="post" action="{% url 'socialaccount_signup' %}">
   {# Friendly welcome message #}
+  {% if username %}
   <h1>{% blocktrans with username=form.username.value %}Welcome to KoboToolbox, {{username}}!{% endblocktrans %}</h1>
+  {% else %}
```
Comment (jnm) [inline kobo/apps/accounts/templates/socialaccount/signup.html:12]: If `username` is defined by the `blocktrans` tag, would this condition ever evaluate to true?

**Review by jnm — CHANGES_REQUESTED** [review]
Thanks for the proposed contribution. I'll follow up about the wording change.

File: kobo/apps/accounts/templates/socialaccount/signup.html [hunk 2]
```diff
@@ -7,7 +7,11 @@
 
 <form class="registration registration--social" id="signup_form" method="post" action="{% url 'socialaccount_signup' %}">
   {# Friendly welcome message #}
+  {% if username %}
```
Comment (paulschreiber) [inline kobo/apps/accounts/templates/socialaccount/signup.html:10]:
```suggestion
  {% if form.username and form.username.value %}
```

File: kobo/apps/accounts/templates/socialaccount/login.html [hunk 3]
```diff
@@ -24,9 +24,9 @@ <h1>{% trans "Set up Single Sign On" %}</h1>
     </button>
   {% else %}
     {# "Log in" #}
-    <h1>{% trans "Log in with Single Sign On" %}</h1>
+    <h1>{% trans "Log in with " %} {{ appname }}</h1>
 
-    <p>{% blocktrans %}Click the button below to log in to KoboToolbox with your {{ appname }} account.{% endblocktrans %}</p>
+    <p>{% blocktrans %}Log in to KoboToolbox with your {{ appname }} account.{% endblocktrans %}</p>
```
[the remark this answers is not in the document]
Comment (paulschreiber) [reply kobo/apps/accounts/templates/socialaccount/login.html:29]: I agree this is redundant. Perhaps we could remove the middle paragraph.

[the remark this answers is not in the document]
Comment (jnm) [reply kobo/apps/accounts/templates/socialaccount/login.html:29]: After some internal discussion, we would like to keep your edits to the first and second lines but change the button to say just "LOG IN". While you're at it, if you wouldn't mind correcting "in to" to be "into", I'd be much obliged. I realize that mistake predated your PR.

[the remark this answers is not in the document]
Comment (paulschreiber) [reply kobo/apps/accounts/templates/socialaccount/login.html:29]: I have changed the button name. I believe "log in to" is correct. See [Stack Exchange discussion](https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to).

[the remark this answers is not in the document]
Comment (jnm) [reply kobo/apps/accounts/templates/socialaccount/login.html:29]: OK, I think the SE accepted answer is mostly a repudiation of "login" as a verb vs. "log in"—which I certainly agree with—but its nod to "log in to + noun" as more correct than "log into + noun" seems to neglect a lot of other valid points raised by the comments and other answers. Let's not spend any more time on it, though :ok_hand:

**Pull request merged**